oaknational/Oak-Web-Application
# Added `display: block;` to fix sticky issues in curric visualiser (iOS)

## Description
Added `display: block;` to fix sticky issues in curric visualiser (iOS)

## Issue(s)

Fixes `CUR-1034`

## How to test

1. Go to https://deploy-preview-2968--oak-web-application.netlify.thenational.academy
2. Safari/iOS curriculum visualiser iOS in small screen
3. Check sticky header works as expected.
